src/go/k8s/apis/redpanda/v1alpha1/cluster_types.go

```diff
@@ -318,7 +318,7 @@ func (r *Cluster) AdminAPIInternal() *AdminAPI {
 // AdminAPIExternal returns internal admin listener
 func (r *Cluster) AdminAPIExternal() *AdminAPI {
 	for _, el := range r.Spec.Configuration.AdminAPI {
-		if !el.External.Enabled {
+		if el.External.Enabled {
```

Comment:
I see you fix it here. Perhaps squash fix with previous commit.

Reply:
ah, okay, let me figure out how to do that :)